ui/src/app/my-workspace/my-workspace.component.ts

```diff
@@ -0,0 +1,71 @@
+import {Component, Inject, LOCALE_ID, OnInit} from "@angular/core"
+import {ManageCollectionComponent} from "../collection/detail/manage-collection.component"
+import {RichSkillService} from "../richskill/service/rich-skill.service"
+import {ToastService} from "../toast/toast.service"
+import {CollectionService} from "../collection/service/collection.service"
+import {ActivatedRoute, Router} from "@angular/router"
+import {Title} from "@angular/platform-browser"
+import {AuthService} from "../auth/auth-service"
+import {TableActionDefinition} from "../table/skills-library-table/has-action-definitions"
+import {ButtonAction} from "../auth/auth-roles"
+import {PublishStatus} from "../PublishStatus"
+import {size} from "lodash"
+
+@Component({
+  selector: "app-my-workspace",
+  templateUrl: "../collection/detail/manage-collection.component.html"
+})
+export class MyWorkspaceComponent extends ManageCollectionComponent implements OnInit {
+
+  constructor(
+    protected router: Router,
+    protected richSkillService: RichSkillService,
+    protected toastService: ToastService,
+    protected collectionService: CollectionService,
+    protected route: ActivatedRoute,
+    protected titleService: Title,
+    protected authService: AuthService,
+    @Inject(LOCALE_ID) protected locale: string
+  ) {
+    super(router, richSkillService, toastService, collectionService, route, titleService, authService, locale)
+    this.uuidParam = "00ff748a-7141-47f2-aaf5-f9b8a992505f"
```

Comment:
Are we providing a fixed UUID? Each workspace collection will still have its own typical UUID. Maybe I misunderstand what I'm seeing?

Reply:
Currently, is static to test the interaction with the new feature, when request to get workspace is ready this will be removed.